src/applications/personalization/account/containers/AccountApp.jsx

```diff
@@ -1,17 +1,29 @@
-import React from 'react';
+import React, { useEffect } from 'react';
 import { connect } from 'react-redux';
 import backendServices from 'platform/user/profile/constants/backendServices';
 import { selectUser, isLOA3 } from 'platform/user/selectors';
 
 import AccountMain from '../components/AccountMain';
 import RequiredLoginView from 'platform/user/authorization/components/RequiredLoginView';
+import localStorage from 'platform/utilities/storage/localStorage';
 import DowntimeNotification, {
   externalServices,
 } from 'platform/monitoring/DowntimeNotification';
 
 import { fetchMHVAccount } from 'platform/user/profile/actions';
 
+import { selectShowProfile2 } from 'applications/personalization/profile-2/selectors';
+
 function AccountApp(props) {
+  useEffect(
+    () => {
+      if (props.redirect) {
+        document.location.replace('/profile/account-security');
```

Comment:
This route currently does not work but it will after https://github.com/department-of-veterans-affairs/vets-website/pull/12607 and https://github.com/department-of-veterans-affairs/devops/pull/6976 are merged.

Reply:
why do we need to do a client side redirect here instead of using the existing redirect functionality in the revproxy?

Reply:
I knew someone would ask this. Can we do a redirect that way based on a Flipper feature flag? The plan is set up a proper 301 redirect when Profile 2.0 is out to 100% of people and we can remove the standalone Account page.

Reply:
Cool that works.